nifi-nar-bundles/nifi-hadoop-bundle/nifi-hdfs-processors/pom.xml

```diff
@@ -78,6 +78,11 @@
             <version>1.15.0-SNAPSHOT</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.github.ben-manes.caffeine</groupId>
+            <artifactId>caffeine</artifactId>
+            <version>2.8.1</version>
```

Comment:
The latest version is `3.0.4`, can you use the latest version?

Reply:
Looks like 3.x requires Java 11:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project nifi-hdfs-processors: Compilation failure: Compilation failure: 
[ERROR] nifi/nifi-nar-bundles/nifi-hadoop-bundle/nifi-hdfs-processors/src/main/java/org/apache/nifi/processors/hadoop/PutHDFS.java:[19,41] error: cannot access Cache
[ERROR]   bad class file: .m2/repository/com/github/ben-manes/caffeine/caffeine/3.0.4/caffeine-3.0.4.jar(com/github/benmanes/caffeine/cache/Cache.class)
[ERROR]     class file has wrong version 55.0, should be 52.0
[ERROR]     Please remove or make sure it appears in the correct subdirectory of the classpath.
```

Reply:
Thanks for pointing that out @greyp9. In that case, what about version `2.9.2`?

Reply:
That seems to work.  Updated.